tests/metrics/collect_mem_consmd.sh

```diff
@@ -128,38 +134,115 @@ function start_collect()
 # It kills the collectl process.
 function stop_collect()
 {
+	echo "Stop collectl"
 	kill "$COLLECT_PID"
 }
 
-# Get the average about a "csv" file produced
-# by collectl tool.
-function get_average()
+# It launches a instance of psstop tool
+# in order to get metrics about proportional
+# set size.
+function start_psstop()
 {
+	echo "Launching psstop"
+	psstop_monitor=$(cat << EOF
```

Comment:
It would be simpler if this script didn't automatically generate another script, but just called another script passing the arguments it needs.